test/v2-migration-tests/src/test/resources/software/amazon/awssdk/v2migrationtests/maven/after/src/main/java/foo/bar/S3.java

```diff
@@ -253,7 +253,7 @@ private void singleBucketArgMethods(S3Client s3, String bucket) {
         s3.getBucketPolicy(GetBucketPolicyRequest.builder().bucket(bucket)
             .build());
         s3.getBucketLocation(GetBucketLocationRequest.builder().bucket(bucket)
-            .build());
+            .build()).locationConstraint().toString();
```

Comment:
Can we update it to return a variable?

Reply:
You mean the test? Its covered it in S3RequestConstructor

https://github.com/aws/aws-sdk-java-v2/blob/157dd1091c66c90aa3ec2ff06a55c4f3af89fb6a/test/v2-migration-tests/src/test/resources/software/amazon/awssdk/v2migrationtests/maven/after/src/main/java/foo/bar/S3RequestConstructor.java#L264